use-cases/school-timetabling/src/main/resources/META-INF/resources/app.js

```diff
@@ -322,15 +275,15 @@ function replaceQuickstartTimefoldAutoHeaderFooter() {
             <span class="navbar-toggler-icon"></span>
           </button>
           <div class="collapse navbar-collapse" id="navbarNav">
-            <ul class="navbar-nav">
+            <ul class="nav nav-pills">
```

Comment:
I absolutely hate that this happens in JS.
Without JS, this simply doesn't happen. Key parts of the UI are not going to be there.

*This is wrong*. (Unfortunately, it is not new.)

Reply:
To summarize the discussion we had:
- it's a trade-off if we eventually want to share the header and footer across quickstarts
- not sharing is problematic, too: in the absence of automated UI tests, it's very easy to make copy-paste mistakes
- I will yet look into the options of sharing the code; this is just a preparatory work